src/test/scala/com/wavesplatform/TransactionGen.scala

```diff
@@ -156,14 +157,14 @@ trait TransactionGen {
     lease = LeaseTransaction.create(sender, amount, fee, feeScale, timestamp, recipient).right.get
     cancelFee <- smallFeeGen
     feeScale2: Short <- feeScaleGen
-  } yield (lease, LeaseCancelTransaction.create(sender, lease.id, cancelFee, feeScale2, timestamp + 1).right.get)
+  } yield (lease, LeaseCancelTransaction.create(recipient, lease.id, cancelFee, feeScale2, timestamp + 1000000).right.get)
```

Comment:
it seems here signer of the `leaseCancelTransaction` should be the singer of the `LeaseTransaction` too?

Reply:
i forgot to change this line back after debugging the unit test related to ```LeaseCancelTransaction``` as well as the ```timestamp + 100000```
changed them back in new commit